modules/compliance-supported-profiles.adoc

```diff
@@ -393,6 +398,11 @@ BSI (Bundesamt für Sicherheit in der Informationstechnik, Germany’s Federal O
 [id="stig-profiles_{context}"]
 == STIG compliance profiles
 
+[IMPORTANT]
+====
+If the `rhcos4-stig` or the `rhcos4-stig-v1r1` profile includes the `rhcos4-service-sshd-disabled` rule, the `sshd` service is disabled and this blocks access to control plane nodes and compute nodes. If you want to keep access to these nodes and exclude a rule from the base profile enforcement, create a `TailoredProfile` object and set the `rhcos4-service-sshd-disabled` annotation in the configuration for the object.
```

Comment:
same as above

Reply:
I am not sure if you still want to update it. How about the below sentences?
Applying auto remediations for any profile(such as rhcos4-stig) using service-sshd-disabled rule will automatically disable the sshd service, which blocks SSH access to control plane and compute nodes. To avoid this and keep SSH access enabled, create a TailoredProfile object and use the disableRules parameter to exclude the rhcos4-service-sshd-disabled rule.